packages/debugger/src/handlers/notebook.ts

```diff
@@ -26,10 +32,34 @@ export class NotebookHandler implements IDisposable {
     this._debuggerService = options.debuggerService;
     this._notebookPanel = options.widget;
     this._cellMap = new ObservableMap<EditorHandler>();
+    this.translator = options.translator || nullTranslator;
```

Comment:
Is the translator actually being passed down when `NotebookHandler` is initialized? It might be it is just not obvious from the diff.

Reply:
Yes, it needs to be threaded down via both the places.